snowpack/src/config.ts

```diff
@@ -407,6 +407,7 @@ function normalizeConfig(_config: SnowpackUserConfig): SnowpackConfig {
       entrypoints: config.optimize.entrypoints ?? 'auto',
       preload: config.optimize.preload ?? false,
       bundle: config.optimize.bundle ?? false,
+      sourcemap: config.optimize.splitting ?? true,
```

Comment:
@yqrashawn  Should this be `config.optimize.sourcemap` instead of `config.optimize.splitting`?

Reply:
Sorry for that. Fixed in #3115